lib/normalize.js

```diff
@@ -32,6 +33,16 @@ var _splitLocator = function (nemo, locatorString) {
 module.exports = function normalize(nemo, _locator) {
   var locator = _locator;
   var normalizedLocator;
+
+  if (locator.using && locator.value && nemo.wd.By[locator.using]) {
+    normalizedLocator = nemo.wd.By[locator.using](locator.value);
+    if(normalizedLocator.constructor === locator.constructor){
+      return locator; //input object was a normalized locator.  Just return it back
+    } else {
+      return normalizedLocator;
+    }
+  }
+
   if (locator.constructor !== String) {
```

Comment:
OK. Partly I've been reacquainting myself with this functionality. I understand now why you have done it this way. The only thing I'd ask in order for us to merge this would be to fix the spacing on line 39 (spacing around if conditional).